### PR TITLE
Add some contributions to our computationally reproducible manuscript

### PR DESCRIPTION
**Pull request by a-paxton**
Thanks for pulling this together (and for your teaching throughout the week), @NelleV ! Here's my first contribution to the manuscript.

### CODE REVIEW

File: doc/manuscript_Rmd/manuscript.Rmd [hunk 1]
```diff
@@ -38,7 +45,7 @@ knitr::opts_chunk$set(
 # Introduction
 
 
-This manuscript is an entirely computationnaly reproducible manuscript. The
+This manuscript is an entirely computationally reproducible manuscript. The
```
Comment (NelleV) [inline doc/manuscript_Rmd/manuscript.Rmd:46]: Thank you so much!

**Review by NelleV — APPROVED** [review]

File: doc/manuscript_Rmd/manuscript.Rmd [hunk 2]
```diff
@@ -7,9 +7,16 @@ author:
 address:
   - code: a
     address: "University of California, Berkeley"
+author:
```
Comment (NelleV) [inline doc/manuscript_Rmd/manuscript.Rmd]: Actually, this doesn't work. You would need to add your name l.6, just under my email address.

File: doc/manuscript_Rmd/manuscript.Rmd [hunk 3]
```diff
@@ -7,9 +7,16 @@ author:
...
+  - name: "Alexandra Paxton"
+    affiliation: b
+    email: "alexandra.paxton@uconn.edu"
+address:
```
Comment (NelleV) [inline doc/manuscript_Rmd/manuscript.Rmd:10]: Same here, but under Berkeley's address.

File: doc/manuscript_Rmd/manuscript.Rmd (hunk 2 above)
Comment (a-paxton) [reply doc/manuscript_Rmd/manuscript.Rmd]: Ah, thanks for the clarification! I think I've fixed the problem.

**Pull request merged**